.github/workflows/Pkg EdFi.Ods.Minimal.Template.PostgreSQL.yml

```diff
@@ -30,10 +30,6 @@ jobs:
     runs-on: windows-latest
 
     steps:
-    - name: Setup .NET
-      uses: actions/setup-dotnet@607fce577a46308457984d59e4954e075820f10a # v3.0.3
-      with:
-          dotnet-version: 3.1.x   
```

Comment:
Removed this step because we need dotnet v6 which comes pre-installed with the base image.